label_studio_ml/examples/watsonx_llm/data_transfer_app.py

```diff
@@ -0,0 +1,163 @@
+                # upload new annotation to watsonx
+                values = tuple([data[key] for key in table_info_keys])
+                insert_command = f"""INSERT INTO {table} VALUES {values}"""
+                print(insert_command)
+                cur.execute(insert_command)
+
+            elif action == "ANNOTATION_UPDATED":
+                # update existing annotation in watsonx by deleting the old one and uploading a new one
+                delete = f"""DELETE from {table} WHERE ID={data["ID"]}"""
+                print(delete)
+                cur.execute(delete)
+                values = tuple([data[key] for key in table_info_keys])
+                insert_command = f"""INSERT INTO {table} VALUES {values}"""
+                print(insert_command)
+                cur.execute(insert_command)
+
+            elif action == "ANNOTATIONS_DELETED":
+                # delete existing annotation in watsonx
+                delete = f"""DELETE from {table} WHERE ID={data["ID"]}"""
+                print(delete)
+                cur.execute(delete)
+
+            conn.commit()
+    except Exception as e:
+        print(e)
+
+def connect_ls():
+    try:
+        client = LabelStudio(
+            base_url = os.getenv("LABEL_STUDIO_URL"),
+            api_key=os.getenv("LABEL_STUDIO_API_KEY")
+        )
+    except Exception as e:
+        print(e)
+    return client
+def get_data(annotation, task, client):
+    """Collect the data to be uploaded to WatsonX.data"""
+    info = {}
+
+    try:
+        users = client.users.list()
+        id = task["id"]
+        annotator_complete = annotation["completed_by"]
+        annotator_update = annotation["updated_by"]
+        annotator_complete = next((x.email for x in users if x.id == annotator_complete), "")
+        annotator_update = next((x.email for x in users if x.id == annotator_update), "")
+        info.update({"ID": int(id), "completed_by": annotator_complete, "updated_by": annotator_update})
+        for key, value in task["data"].items():
+            if isinstance(value, List):
+                value = value[0]
+            elif isinstance(value, str) and value.isnumeric():
+                value = int(value)
+
+            if isinstance(value, str):
+                value = value.strip("\"")
+            info.update({key: value})
+
+
+        for result in annotation["result"]:
+            print(result)
+            val_dict_key = list(result["value"].keys())[0]
+            value = result["value"][val_dict_key]
+            key = result["from_name"]
+            if isinstance(value, List):
+                value = value[0]
+            elif isinstance(value, str) and value.isnumeric():
+                value = int(value)
+
+            if isinstance(value, str):
+                value = value.strip("\"")
+            info.update({key: value})
+
+        print(f"INFO {info}")
+        return info
+    except Exception as e:
+        print(e)
```

Comment:
must be more descriptive message here, including the traceback `traceback.format_exc()`